Druid/DRPrimitiveCompilerCompiler.class.st

```diff
@@ -19,10 +19,10 @@ DRPrimitiveCompilerCompiler class >> compilePrimitive: primitiveName from: anInt
 ]
 
 { #category : #'instance creation' }
-DRPrimitiveCompilerCompiler class >> forInterpreter: aDRInterpreterCompiler [ 
+DRPrimitiveCompilerCompiler class >> forInterpreter: anAbstractInterpreter [ 
 
 	^ self new
-		interpreter: aDRInterpreterCompiler;
+		interpreter: anAbstractInterpreter;
 		yourself
 ]
```

Comment:
The variable should be `anInterpreter`. We will never get an instance of an abstract interpreter. Because if abstract it cannot have instances :)